data/data/bootstrap/bootstrap-in-place/files/opt/openshift/bootstrap-in-place/bootstrap-in-place-post-reboot.sh

```diff
@@ -24,13 +24,33 @@ function signal_bootstrap_complete {
   done
 }
 
+function release_cvo_lease {
+  if [ ! -f /opt/openshift/release_cvo_lease.done ]
+  then
+    until [ "$(oc get leases -n openshift-cluster-version version | grep -c "version")" -eq 0 ];
+    do
+      echo "Deleting openshift-cluster-version version lease"
+      oc delete leases -n openshift-cluster-version version || sleep 5
+    done
+    until [ "$(oc get cm -n openshift-cluster-version version | grep -c "version")" -eq 0 ];
+    do
+      echo "Deleting openshift-cluster-version version cm"
+      oc delete cm -n openshift-cluster-version version || sleep 5
+    done
+    touch /opt/openshift/expedite_bootstrapping.done
+  fi
+}
+
 function restart_kubelet {
-  echo "Restarting kubelet"
-  until [ "$(oc get pod -n openshift-kube-apiserver-operator --selector='app=kube-apiserver-operator' -o jsonpath='{.items[0].status.conditions[?(@.type=="Ready")].status}' | grep -c "True")" -eq 1 ];
+  echo "Waiting for kube-apiserver-operator"
+  until [ "$(oc get pod -n openshift-kube-apiserver-operator --selector='app=kube-apiserver-operator' -o jsonpath='{.items[*].status.conditions[?(@.type=="Ready")].status}' | grep -c "True")" -eq 1 ];
   do
     echo "Waiting for kube-apiserver-operator ready condition to be True"
+    oc get --raw='/readyz' &> /dev/null || echo "Api is not available"
```

Comment:
Is this temporary debugging, or are  you intending to dump `/readyz` output to stdout long-term?

Reply:
I'd like to keep it, it's not dumping the  /readyz output to stdout (`&> /dev/null`) it will just output the message as long the API isn't available while we wait for kube-apiserver-operator